src/ontology/uberon-edit.obo

```diff
@@ -219746,11 +219746,9 @@ is_a: UBERON:6007248 ! insect chorionic specialization
 
 [Term]
 id: UBERON:6000094
-name: clypeo-labral anlage in statu nascendi
-def: "." [FBC:VH]
-synonym: "Asn Cly" RELATED [FBC:DOS]
-is_a: UBERON:6005413 ! insect anlage in statu nascendi
-relationship: part_of FBbt:01000119
+name: obsolete clypeo-labral anlage in statu nascendi
+comment: No need for such a specific term in Uberon.
```

Comment:
could you put a link to this PR in the comment please - think we are trying to do that with obsoletions now.

Reply:
I hope you mean “a link to the original ticket” (here #2154) and not a link to the _PR_… Otherwise that would mean that for any obsoletion one need to first edit the ontology file to obsolete the term (without adding a link to a PR that does not yet exist), create the PR, get the PR number, edit the ontology file again to add the link, and update the PR… I quite don’t see the benefit of such a cumbersome approach.

Reply:
oh yeah if there is a ticket, that's perfect :) its mainly to keep discussion on why the obsoletion is done - I saw the write up on this PR so thought it was all here.